### PR TITLE
Update the AWS dockercfg generator

### PR DESCRIPTION
**Pull request by mlocher**
- Update to Alpine 3.4
- Specify the AWS CLI version via an environment variable (and update to the latest version)
- Switch from ADD to COPY
- Test by pushing to a AWS ECR registry on the Codeship Sandbox account.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 
-ADD aws_docker_creds.sh /
+RUN \
+  apk --no-cache add \
+    py-pip \
+    python && \
+  pip install awscli==${AWS_CLI_VERSION} && \
+  mkdir -p "${HOME}/.aws"
+
+COPY aws_docker_creds.sh /
```
Comment (rheinwein) [inline Dockerfile:22]: ❤️

**Comment by rheinwein** [comment]
LGTM

**Comment by rheinwein** [comment]
bummer that 3.4 is so much bigger than 3.3 :(

**Comment by mlocher** [comment]
Yeah, the new Docker releases are a lot bigger than old ones :(

**Pull request merged**